docs/content/node-operation/node-roles.mdx

```diff
@@ -52,3 +52,11 @@ Verification nodes are required to stake a minimum of 135,000 FLOW to be a confi
 Access nodes act as a proxy to the Flow network. The Access node routes transactions to the correct Collection node and routes state queries to available nodes (while likely caching state to answer queries in a timely manner in the future). Dapps and user agents can submit their transactions to any Access node or run their own if they can't find a service provider they're happy with.
 
 Access nodes are required to stake a 0 FLOW to be a confirmed node operator.
```

Comment:
Is 'a 0 FLOW' correct here?

Reply:
@franklywatson that doesn't sound right to me as well. From git history it looks like @vishalchangrani introduced this line 3 months ago and may be able to confirm if this was intended or just a typo.

Source: https://github.com/onflow/flow/blame/11c8350d2e9a96d1b7d6974a28d4e31cec6419ed/docs/content/node-operation/node-roles.mdx#L54

Reply:
this is intended - we do ask the access node operators to stake with 0 FLOW since Access nodes are staked but don't have to put up any minimum and do not receive any staking rewards either. But yes they still have to go into Flow port and stake with 0 Flow.
Gramatically though the sentence could be written better - Access nodes are required to stake with 0 FLOW (but still stake) to be confirmed to be a node operator.